scripts/example_script.py

```diff
@@ -0,0 +1,35 @@
+from setup.authentication import get_auth_header
+import setup.constants as constants
+from setup.email import create_email_integration
+from setup.healthcheck import check_health
+from setup.mailchimp_connector import create_mailchimp_connector
+from setup.postgres_connector import create_postgres_connector
+from setup.user import create_user
+
+
+print("Running an example Fides configuration script...")
+
+try:
+    check_health()
+except RuntimeError:
+    print(
+        f"Connection error. Please ensure Fides is healthy and running at {constants.FIDESOPS_URL}."
+    )
+
+auth_header = get_auth_header()
```

Comment:
I'm not a huge fan of this pattern of passing the `auth_header` into every function. It makes more sense than having the script fetch a new one each time, and absent any class structure to attach the header to this appears to be the next best thing.